app/src/main/java/net/gsantner/markor/format/zimwiki/ZimWikiHighlighter.java

```diff
@@ -0,0 +1,145 @@
+/*#######################################################
+ *
+ *   Maintained by Gregor Santner, 2018-
+ *   https://gsantner.net/
+ *
+ *   License of this file: Apache 2.0 (Commercial upon request)
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+#########################################################*/
+package net.gsantner.markor.format.zimwiki;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.text.InputFilter;
+import android.text.Spannable;
+
+import net.gsantner.markor.format.ListHandler;
+import net.gsantner.markor.model.Document;
+import net.gsantner.markor.ui.hleditor.Highlighter;
+import net.gsantner.markor.ui.hleditor.HighlightingEditor;
+import net.gsantner.markor.util.AppSettings;
+
+import java.util.regex.Pattern;
+
+import other.writeily.format.zimwiki.WrZimWikiHeaderSpanCreator;
+
+public class ZimWikiHighlighter extends Highlighter {
+    private final boolean _highlightBiggerHeadings;
+    public final String _fontType;
+    public final Integer _fontSize;
+
+    private static final int COLOR_HEADING = 0xff4e9a06;
+    private static final int HIGHLIGHT_BACKGROUND_COLOR_LIGHT_MODE = 0xffffff00;
```

Comment:
* feedback - gs2

markor dark theme: Color for highlighting doesn't work in this case. I suggest to use a darkish-yellow/brown in that case. (Instead of keeping yellow for dark mode, and adjusting text foreground color, because that costs performance twice)

Thought: `0xff_FFA062`, kinda yellow-orang'ish-brown'ish

![screenshot-2020-11-14__13-19-52](https://user-images.githubusercontent.com/6735650/99146848-24cac200-267c-11eb-977f-17b83f59bb8a.png)

Reply:
The orange color is fine, changed it to use it exclusively for both light and dark mode.